cmsplugin_cascade/templates/cascade/admin/change_form.html

```diff
@@ -2,7 +2,7 @@
 {% load admin_static admin_urls i18n %}
 
 {% block field_sets %}
-	<h1>{{ plugin_title }}</h1>
+	<!--<h1>{{ plugin_title }}</h1>-->
```

Comment:
Thanks Jacob for the review,

I did is to avoid this

![image](https://cloud.githubusercontent.com/assets/488513/20359838/db3a3ac6-ac31-11e6-9d6e-85ea98c0a870.png)

in CKEditor and get this

![image](https://cloud.githubusercontent.com/assets/488513/20359850/e7299e58-ac31-11e6-8651-afba81f44180.png)

Patrick